programmers/3/3_2.js

```diff
@@ -0,0 +1,10 @@
+//자릿수 더하기
+function solution(n)
+{
+    let answer = 0;
+    let m=String(n);
+    for(let i=0; i<m.length; i++){
+        answer+=parseInt(m[i]);
```

Comment:
```suggestion
    const m = String(n);
    for(let i = 0; i < m.length; i++) {
        answer += parseInt(m[i]);
```
공백 컨벤션, m은 나중에 변하지 않는 변수로 쓰시고 계시니 const 로 선언 해주세요 
변수명도 m 보다는 누구나 의도를 알기 쉽게 직관적으로 작성해주세요